README.md

```diff
@@ -1,2 +1,70 @@
-# service
+# Eik HTTP Service
+
 Eik REST API - As a standalone running HTTP service
+
+## Installation
+
+```
+npm install @eik/service
+```
+
+## Usage
+
+This server can either be run as a Node executable, or as a Fastify plugin.
+
+```sh
+npx @eik/service
+```
+
+For a production setup, the Fastify plugin method is recommended.
+
+```js
+import fastify from 'fastify';
+import Service from '@eik/service';
+import Sink from '@eik/sink-gcs';
+
+// Set up the Google Cloud Storage sink
+// https://github.com/eik-lib/sink-gcs?tab=readme-ov-file#example
+const sink = new Sink({
+    credentials: {
+        client_email: 'a@email.address',
+        private_key: '[ ...snip... ]',
+        projectId: 'myProject',
+    },
+});
```

Comment:
```suggestion
import GcsSink from '@eik/sink-gcs';

// Set up the Google Cloud Storage sink
// https://github.com/eik-lib/sink-gcs?tab=readme-ov-file#example
const sink = new GcsSink({
    credentials: {
        client_email: 'a@email.address',
        private_key: '[ ...snip... ]',
        projectId: 'myProject',
    },
});
```
Naming the default import makes it easier to read and grok what's happening I feel.